tsweb/tsweb.go

```diff
@@ -295,23 +309,52 @@ func (f ReturnHandlerFunc) ServeHTTPReturn(w http.ResponseWriter, r *http.Reques
 // Handled requests are logged using opts.Logf, as are any errors.
 // Errors are handled as specified by the Handler interface.
 func StdHandler(h ReturnHandler, opts HandlerOptions) http.Handler {
-	if opts.Now == nil {
-		opts.Now = time.Now
-	}
 	if opts.Logf == nil {
-		opts.Logf = logger.Discard
+		return Handler(h, opts)
 	}
-	return retHandler{h, opts}
+	return LogHandler(Handler(h, opts), opts)
 }
 
-// retHandler is an http.Handler that wraps a Handler and handles errors.
-type retHandler struct {
-	rh   ReturnHandler
+// LogHandler's handled requests are logged using opts.Logf, as are any errors.
```

Comment:
I feel like this is missing the first sentence saying what this does. 

Also, try to avoid passive voice and use the active voice.

Maybe like:

    // LogHandler returns an http.Handler that logs to opts.Logf.
    // It logs both success and failing requests.

Reply:
Updated: https://github.com/tailscale/tailscale/blob/18ff30b4c8b61eda1234f5884e08a8fcb7b00e3f/tsweb/tsweb.go#L318-L322